app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,16 @@
+class CommentsController < ApplicationController
+
+  def create
+    @company = Company.find(params[:company_id])
+    @comment = Comment.new(comment_params)
+    @comment.job_id = params[:job_id]
```

Comment:
this is a perfect place to refactor to leverage your associations like we discussed today!